storagenode/storagenodedb/database.go

```diff
@@ -232,22 +209,35 @@ func (db *DB) Close() error {
 		db.ndb.Close(),
 		db.adb.Close(),
 
-		db.versionsDB.Close(),
-		db.v0PieceInfoDB.Close(),
-		db.bandwidthDB.Close(),
-		db.ordersDB.Close(),
-		db.pieceExpirationDB.Close(),
-		db.pieceSpaceUsedDB.Close(),
-		db.reputationDB.Close(),
-		db.storageUsageDB.Close(),
-		db.usedSerialsDB.Close(),
-		db.satellitesDB.Close(),
+		db.closeDatabases(),
 	)
 }
 
-// Versions returns the instance of the versions database.
-func (db *DB) Versions() SQLDB {
-	return db.versionsDB
+// closeDatabases closes all the SQLite database connections and removes them from the associated maps.
+func (db *DB) closeDatabases() error {
+	var err error
```

Comment:
Use `errs.Group` this way it won't create a large chain of error wrapping.

Reply:
Done!